sale_commission_agent_restrict/models/sale.py

```diff
@@ -9,6 +11,7 @@ class SaleOrder(models.Model):
         compute="_compute_commission_total",
         store=True,
     )
+    partner_id_domain = fields.Char(compute="_compute_partner_id_domain")
```

Comment:
can't we use a `Serialized` field instead? Just asking... I'm not sure it works w/ the domain widget ;)